src/pages/Options/Earn/Vesting/Vesting.tsx

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Schedule from './Schedule';
 import YourTransactions from './Transactions';
 import Vest from './Vest';
 import Info from './Info';
 
 const Vesting: React.FC = () => {
+    useEffect(() => {
+        window.scrollTo(0, 0);
+    }, []);
+
```

Comment:
Can we move this scroll to parent component and do it when user changes tab? Its repeating a lot.